HTML+CSS+JS/Cards/Card Pop up on hover/README.md

```diff
@@ -3,6 +3,10 @@
 # GIF for Task:
 
 
+<img align="center" height="500" width="800" src="https://media.giphy.com/media/9pKgy1jRHMxB06OEux/giphy.gif">
```

Comment:
This won't center align, you did the same in previous component that is why I had added a div with align= center and it got centered so remember this point